### DISCUSSION TITLE
Mac M1 Terminal couldn't connect to homebrew

### DISCUSSION THREAD

**Discussion by mirafei** [Homebrew/discussions#3744] [Getting started]
I tried to install homebrew on my m1 mac and it gave me this response.
[ Failed to connect to raw.githubusercontent.com port 443 after 101 ms: Connection refused](curl: (7) Failed to connect to raw.githubusercontent.com port 443 after 101 ms: Connection refused)

**Comment by SMillerDev** [comment]
Your internet connection appears to be blocking the connection to GitHub, you'll have to figure out why for yourself.

**Comment by mustaphamushi** [comment]
@mirafei Did you figure out the problem? I have the same issue and I have tried all suggestions on the internet.

**Comment by WilsonHuha** [comment]
Same issue here, however another way around, you can 'try' copy the install.sh and execute it locally on your Macbook M1. Life will be easy. Hope it helps. Nice day.
